oat-sa/tao-core
# Fix/tao 7265 datetimepicker active hover style

Linked to https://oat-sa.atlassian.net/browse/TAO-7265

All hover/focus/active styles from the date picker weren't applied  because I broke it due to a bug in my auto-formatter.

The style is now back to it's initial state.

How to test : 
 - Use `<Tab>` or `<S-Tab>` in a picker field to give the focus to the _toggle_ button
 - without the PR no style indicates the button has the focus
 - with the PR you'll see a blue border around it
